phpunit.xml.dist

```diff
@@ -57,4 +57,9 @@
 	<testsuite name="backend-mixed">
 		<directory>tests/Backend/Mixed</directory>
 	</testsuite>
+
+	<testsuite name="storage-types-feature">
```

Comment:
As I wrote at KBC PR. I will put those tests on normal worksuites. It's a little bit tricky to create a new testsuite. I think I'm sure about that how those tests works, so we can have a call if you want to and discuss the best solution.